File: backend-python/src/app/models/message.py
from datetime import datetime
from uuid import UUID, uuid4

from sqlalchemy import DateTime, String
from sqlalchemy.orm import Mapped, mapped_column

from .base import Base


class Message(Base):
    __tablename__ = "messages"

    id: Mapped[UUID] = mapped_column(primary_key=True, default=uuid4)
    content: Mapped[str] = mapped_column(String(255), default="Hello World", nullable=False)
    created_at: Mapped[datetime] = mapped_column(DateTime, default=datetime.utcnow, nullable=False) 

File: backend-kotlin/src/main/kotlin/brex/interview/model/Message.kt
package brex.interview.model

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table
import java.time.LocalDateTime
import java.util.UUID

@Entity
@Table(name = "messages")
data class Message(
    @Id
    val id: UUID? = null,

    @Column(nullable = false)
    val content: String = "Hello World",

    @Column(nullable = false)
    val createdAt: LocalDateTime? = null
)
